packages/algob/src/internal/core/config/config-loading.ts

```diff
@@ -56,16 +57,19 @@ export async function loadConfigAndTasks (
   const netname = runtimeArgs?.network;
   if (netname !== undefined) {
     const net = cfg.networks[netname];
-    if (net !== undefined) { await _loadKMDAccounts(net); }
+    if (net?.kmdCfg !== undefined) {
+      const kmdOp = new KMDOperator(createKmdClient(net.kmdCfg));
+      await _loadKMDAccounts(net, kmdOp);
+    }
   }
 
   return cfg;
 }
 
 // loads KMD accounts if the net.kmdCfg is specified and merges them into net.accounts
-async function _loadKMDAccounts (net: NetworkConfig): Promise<void> {
+export async function _loadKMDAccounts (net: NetworkConfig, kmdOp: KMDOperator): Promise<void> {
```

Comment:
If you export this function, then please rename it. Exported functions should have `_` prefix. You can just remove the `_`.

Maybe we can add this function to the KMDOperator?

Reply:
Okay, will rename. Also, i assumed that this function is for internal use only. Right?
If we can move this, then i suggest to move to the `KmdOperator` class to a new file in `src/lib/kmd-operator.ts` (similar to `algo-operator.ts`).
Currently i am updating the function. Will move according to furthur discussion.

Reply:
yes.